packages/gestalt/package.json

```diff
@@ -47,4 +47,4 @@
     "last 2 versions",
     "iOS 8"
   ]
-}
+}
```

Comment:
this change is weird, maybe an editor thing?

Reply:
Fixed, I needed to setup editorconfig in vscode